packages/loader/container-loader/src/container.ts

```diff
@@ -1700,10 +1700,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.collabWindowTracker?.stopSequenceNumberUpdate();
         return this._deltaManager.submit(type, contents, batch, metadata);
     }
-
+    private readonly ops: any[] = [];
     private processRemoteMessage(message: ISequencedDocumentMessage) {
         const local = this.clientId === message.clientId;
-
+        this.ops.push(message);
```

Comment:
Same feedback here - what's the purpose of this.ops? As this PR stands, change in this file should be removed (as it does nothing other than increase memory consumption).

Reply:
it was for testing, i will comment out for now until the main logic is approved and no test fails